AEPCore/Sources/eventhub/EventHub.swift

```diff
@@ -22,26 +22,36 @@ public typealias EventPreprocessor = (Event) -> Event
 /// Responsible for delivering events to listeners and maintaining registered extension's lifecycle.
 final class EventHub {
     private let LOG_TAG = "EventHub"
-    private let eventHubQueue = DispatchQueue(label: "com.adobe.eventHub.queue")
+    private let identifier: SDKInstanceIdentifier
+    private let logger: Logger
+    private let extensionServiceProvider: ExtensionServiceProvider
+    private let eventHubQueue: DispatchQueue
+    private let eventQueue: OperationOrderer<Event>
     private var registeredExtensions = ThreadSafeDictionary<String, ExtensionContainer>(identifier: "com.adobe.eventHub.registeredExtensions.queue")
     private let eventNumberMap = ThreadSafeDictionary<UUID, Int>(identifier: "com.adobe.eventHub.eventNumber.queue")
     private let responseEventListeners = ThreadSafeArray<EventListenerContainer>(identifier: "com.adobe.eventHub.response.queue")
     private var eventNumberCounter = AtomicCounter()
```

Comment:
Having each EventHub maintain its own eventNumberCounter will simplify debugging. Otherwise, event numbers and shared state versions would become unpredictable, even for related events.